data/wanderer/wanderers start.txt

```diff
@@ -179,6 +180,217 @@ mission "Wanderers: Hai Diplomat"
+			choice
+				`	"You said you were... Skiorologists?"`
+				`	"My only interest is translation. Will you do that for me?"`
+					goto "get to the point"
+			`	"As Skiorologists, we study the development of the Hai, in the same way that Anthropologists study humans. How the Hai interact with other species is a major part of what makes the Hai... Hai. We have researched past interactions, but analyzing an ongoing interaction is a dream come true.`
+			label "get to the point"
+			`	"In return for introducing us to this new species, we'll be happy to translate for you. As we're already prepared to leave for an expedition, we can leave as soon as you meet us in the spaceport."`
+			choice
+				`	"Sure! I'll meet you at the spaceport."`
+					accept
+				`	"No, I'd rather wait for the Hai government to provide an ambassador."`
+					decline
+
+
+
+mission "Wanderers: Diplomacy (Before Hai Reveal)"
+	name "Wanderers: Diplomacy"
+	description "Travel to the Wanderer homeworld with a Hai ambassador who can help communicate with them."
+	source Allhome
+	destination "Vara K'chrai"
+	clearance
+	passengers 1
+	blocked `You arrive at the spaceport, but you suddenly realize you have too few bunks for Professor Andelena and her student.`
+	to offer
+		has "Wanderers: Professor Available: done"
+		not "Wanderers: Diplomacy: active"
+		not "Wanderers: Diplomacy: done"
+		not "Wanderers: Sayari Available After Hai Reveal: offered"
+	on offer
+		fail "Wanderers: Meet Professor"
+		conversation
+			`As you enter the spaceport, a group of armed Hai in uniforms approach you. The uniforms match the one from the Hai that refused to help you on Hai-home.`
+			`	One of them addresses you. "Are you looking for Professor Andelena?"`
+			choice
+				`	"Yes."`
+					goto yes
+				`	"Is she in some kind of trouble?"`
+				`	"I've never heard that name in my life."`
+					decline
+			`	"Neither she nor you are in trouble. We just want you to help us clear up some matters."`
+				goto question
+			label yes
+			`	"There has been an incident and some miscommunications. We want you to help us clear up some matters."`
+			label question
+			choice
+				`	"Lead on."`
+				`	"No, I think I'll be on my way."`
+					decline
+			`	The armed Hai escort you to a secure room, where a worried Professor Andelena looks at her student, Eenap, who is in handcuffs. An stern-looking Hai woman in a similar uniform to the others seems to be in charge, and glares at Eenap before addressing you.`
+			`	"Your efforts to find an interpreter have stumbled upon a suspected Unfettered spy who tried to transmit details about you and your ship to the Unfettered." She glares again at Eenap, who parries with a defiant look. "Fortunately, we shut down the operation before it got too far; you need not worry about your safety in this matter.`
+			`	"However, we are concerned about the level of interest the Unfettered have in these aliens the head of our diplomatic corps wants to send an ambassador."`
```

Comment:
I think I need to move their offer of an interpreter to another mission. It should go like this:

1. Interpreter refused on Hai-home.
2. Wait.
3. Professor wants to see you.
4. Professor's plans fall apart when student is arrested. (That's the part you commented about.)
5. Wait.
6. Message to visit a spaceport somewhere for an interpreter. (I'll pick a planet that Hai Reveal doesn't touch much.)

Reply:
I've added the second step, and the Hai are quite disgruntled about *needing* to send an interpreter. Captain Last made a thorough mess of things by not displaying patience nor respecting privacy. Because of this, the low-level diplomats wanted to send somebody else, but the head of the Hai diplomatic corps (Sayari) overruled them. Sayari chose Captain Last and Irayas for the mission.